portal/src/test/scala/cromwell/pipeline/service/UserServiceTest.scala

```diff
@@ -0,0 +1,69 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dao.repository.UserRepository
+import cromwell.pipeline.datastorage.dto.user.DeactivateUserRequestByEmail
+import cromwell.pipeline.datastorage.dto.{ User, UserDeactivationByEmailResponse, UserDeactivationByIdResponse, UserId }
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.concurrent.ScalaFutures._
+import org.scalatest.{ Matchers, WordSpec }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class UserServiceTest extends WordSpec with Matchers with MockFactory {
```

Comment:
When tests should work with `ExecutionContext` you should use `AsyncWordSpec` instead of usual one.
It provides it's own execution context and make things simpler

Reply:
WordSpec was replaced with AsyncWordSpec in both UserServiceTest and UserControllerItTest